src/evaluator.ts

```diff
@@ -224,7 +225,7 @@ class Evaluator {
             test,
 
             // All of these are removed in python and script providers, but every Javascript provider gets them
-            logger,
+            logger: logger as winston.Logger,
```

Comment:
is there a way to make the StrictLogger type satisfy winston.Logger so you don't need to cast it?